lib/iris/plot.py

```diff
@@ -845,7 +845,9 @@ def _replace_axes_with_cartopy_axes(cartopy_proj):
                 ylabel=ax.get_ylabel(),
             )
         else:
+            position = ax.get_position(original=True)
```

Comment:
I'd have thought that if the axes have been moved we'd want to keep that change in the new axes?

Reply:
I have to admit that, before @QuLogic's comment, I hadn't realised an Axes had more than one position.  From the [docs](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.set_position.html#matplotlib.axes.Axes.set_position) it seems the only way they would be different is if you have set an aspect ratio.  Cartopy will set its own aspect ratio to make the map look sensible.  So I think it does make sense to override any "active" position and revert to the "original" position.